src/features/environmentCreate/components/Specification/SpecificationCreate.tsx

```diff
@@ -129,18 +166,41 @@ export const SpecificationCreate = ({ onCreateEnvironment }: any) => {
     >
       <Box>
         {show ? (
-          <CodeEditor
-            code={formatCode(channels, requestedPackages, environmentVariables)}
-            onChangeEditor={onUpdateEditor}
-          />
+          <>
+            <FormControl sx={{ m: 1, minWidth: 120 }}>
+              <Select
+                value={specificationType}
+                onChange={onUpdateSpecificationType}
+                displayEmpty
+              >
+                <MenuItem value="specification">specification</MenuItem>
+                <MenuItem value="lockfile">unified lockfile</MenuItem>
```

Comment:
MenuItem values here match those we show in the admin interface (see the companion conda-store PR).